CONTRIBUTING.md

```diff
@@ -48,6 +48,13 @@ To do so, make sure that the libraries from [requirements.testenv.txt](requireme
 
 Run [`graphics/setup-sandbox`](graphics/setup-sandbox) script to set up a test repo under `~/machete-sandbox` with a remote in `~/machete-sandbox-remote`.
 
+## Regenerate the GIF in README.md
+
+1. Generate sandbox repositories as shown above.
+1. `cd` into `~/machete-sandbox`.
+1. Install [asciinema](https://github.com/asciinema/asciinema), [asciicast2gif](https://github.com/asciinema/asciicast2gif) and their dependencies.
+1. Run [`graphics/generate-asciinema-gif`](graphics/setup-sandbox) (It will print the location of the generated GIF file)
+1. Replace existing GIF [`graphics/discover-status-traverse.gif`](graphics/discover-status-traverse.gif) with the new one.
 
```

Comment:
While you're here, you can remove `graphics/logo.png` (retained for a moment to keep links valid, no longer needed now; `graphics/logo/` keeps all logo versions)